packages/docusaurus-theme-classic/src/options.ts

```diff
@@ -210,6 +214,15 @@ const DocsVersionDropdownNavbarItemSchema = NavbarItemBaseSchema.append({
   dropdownActiveClassDisabled: Joi.boolean(),
   dropdownItemsBefore: Joi.array().items(DropdownSubitemSchema).default([]),
   dropdownItemsAfter: Joi.array().items(DropdownSubitemSchema).default([]),
+  versions: Joi.alternatives().try(
+    Joi.array().items(Joi.string().min(1)),
+    Joi.object<PropVersionItems>().pattern(
+      Joi.string().min(1),
+      Joi.object<PropVersionItem>({
+        label: Joi.string().min(1),
+      }),
+    ),
+  ),
```

Comment:
can you add some basic tests in `packages/docusaurus-theme-classic/src/__tests__/options.test.ts` ?

We are not covering everything properly but adding tests for Joi schema is good. Joi often has surprisingly weird behaviors.